internal/data/monitor_mock.go

```diff
@@ -0,0 +1,26 @@
+// The functions in this file are used to mock the database connection and the database model.
```

Comment:
For each entity in the database, the idea is develop the mock version, that will be useful in the tests